app/uk/gov/hmrc/nationaldutyrepaymentcenter/connectors/AmendCaseConnector.scala

```diff
@@ -39,19 +39,23 @@ class AmendCaseConnector @Inject()(
 
   override val kenshooRegistry: MetricRegistry = metrics.defaultRegistry
 
-  val url = config.eisBaseUrl + config.eisAmendCaseApiPath
+  val url: String = config.eisBaseUrl + config.eisAmendCaseApiPath
 
   def submitAmendClaim(request: EISAmendCaseRequest, correlationId: String)(implicit hc: HeaderCarrier
   ): Future[EISAmendCaseResponse] = {
     monitor(s"ConsumedAPI-eis-pega-amend-case-api-POST") {
       http.POST[EISAmendCaseRequest, EISAmendCaseResponse](
         url,
         request,
-        eisApiHeaders(correlationId, config.eisEnvironment, config.eisAuthorizationToken)
+        EISApiHeaders(correlationId, config.eisEnvironment, config.eisAuthorizationToken) ++ MDTPTracingHeaders(hc)
```

Comment:
Observation.  Not a fan of capitalised method names.  Looks like you are constructing two objects above.

Reply:
Updated